grpc-api/server/docker/build.gradle

```diff
@@ -113,4 +115,29 @@ dockerCreateDockerfile.dependsOn grpcApiBase
 assemble.dependsOn dockerBuildImage
 dockerBuildImage.outputs.upToDateWhen { t -> return Docker.isImageUpToDate(t) }
 
+// Create a simple file for docker/build-push-action to reference, tag, and publish
+tasks.register('publishDockerfile', Dockerfile) {
+  def input = dockerBuildImage.imageIdFile
+  def output = layout.buildDirectory.file('publish/Dockerfile')
```

Comment:
gitignore this, and maybe mark it for clean to mop up?

Reply:
It should already be ignored, given it's under the build directory, I think.